portal/apps/org.wso2.carbon.iam.admin-portal.app/src/test/java/org/wso2/carbon/iam/adminportal/actionobject/AddNewUserPageAction.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.iam.adminportal.actionobject;
+
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.wso2.carbon.iam.adminportal.pageobject.AddNewUserPage;
+
+public class AddNewUserPageAction extends AddNewUserPage {
+
+    WebDriver webDriver = null;
+
+    public AddNewUserPageAction(WebDriver driver) {
+        super(driver);
+        webDriver = driver;
+    }
+
+    public boolean addNewUser(String username, String password, String domain, String confirmPassword) {
+        Select domainDropdown = new Select(getDomain());
+        domainDropdown.selectByVisibleText(domain);
+        getUsername().sendKeys(username);
+        Select verificationSelectordropdown = new Select(getSelectMethod());
+        verificationSelectordropdown.selectByVisibleText("With password");
```

Comment:
With internationalization this text would be changed to something else. We will need address that as same meaning  will be visible with different locales.